runtime/src/main/java/org/corfudb/runtime/object/transactions/AbstractTransactionalContext.java

```diff
@@ -122,7 +121,7 @@
     private final WriteSetInfo writeSetInfo = new WriteSetInfo();
 
     @Getter
-    private final ReadSetInfo readSetInfo = new ReadSetInfo();
+    private final ConflictSetInfo readSetInfo = new ConflictSetInfo();
```

Comment:
this naming is going to be confusing. I am not saying the previous situation was better.. but still: ConflictSet is for conflicts, but readSet may not actually contain conflicts in the case of write-write transactions.

AccessSetInfo?